doc/installationguide.md

```diff
@@ -98,6 +98,7 @@ The following table shows the accepted environment variables, as well as the con
 
```

Comment:
Please add an entry to CHANGES_NEXT_RELEASE regarding the new functionality this PR implements, eg:

```
- Add: HTTPS suport (if IOTA_CB_URL is enabled) (#578)
```

Reply:
Fixed in a847c8ba672f20c0e84ecec93b9d4feeedab4090

(@chicco785 this time I'm adding the Fixed in mark but, for future PRs, please do it yourself... Reviewers need this kind of marks in order to check when and how each comment has been addressed, expecially in complex PRs where the number of comments is high).